### PR TITLE
Management Commands for Caching Equipment and Rooms

### PR DESCRIPTION
**Pull request by ChristopherHammond13**
## What does this PR do?
Adds caching for Equipment and Rooms

## ✅ Pull Request checklist

- [ ] Is this code complete?
- [ ] Are tests done/modified?

## 📃 Link to PR updating documentation (if applicable)

https://github.com/uclapi/apiDocs/pulls

## 🚨 Is this a breaking change for API clients?
No

## :squirrel: Deploy notes
Need to migrate `gencache` then immediately update the cache on all three endpoints

### CODE REVIEW

**Comment by ChristopherHammond13** [comment]
Changes that need to be done:
1) I'd like the cache update command to do all three endpoints back to back if possible so that it can then switch bucket.
2) The field sizes are all too large. They should be set to the sizes of the fields in the Oracle database to save memory: remember, this is all going into RAM.

File: backend/uclapi/roombookings/management/commands/create_rooms.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    bookabletype=room[10],
+                    roomclass=room[11],
+                    zone=room[12],
+                    webview=room[13],
+                    automated=room[14],
+                    capacity=room[15],
+                    category=room[16]
+                )
+                room_cache.save()
+            except IndexError:
```
Comment (wilhelmklopp) [inline backend/uclapi/roombookings/management/commands/create_rooms.py]: this seems like it will cause trouble

Comment (Faiz) [inline backend/uclapi/roombookings/management/commands/create_rooms.py]: why?

Comment (wilhelmklopp) [inline backend/uclapi/roombookings/management/commands/create_rooms.py]: There shouldn't ever be an index error. If there is one, something is wrong.
This suppresses the warning without fixing the cause, right?

Comment (jermenkoo) [inline backend/uclapi/roombookings/management/commands/create_rooms.py]: L18 already starts from the 2nd (index 1) element, why do we need this then?

File: backend/uclapi/roombookings/management/commands/create_rooms.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand
+from roombookings.models import RoomCache
+
+
+class Command(BaseCommand):
+
+    help = 'Clone room data to gencache'
+
+    def handle(self, *args, **options):
+        with open('roombookings/management/commands/data/rooms.csv') as f:
```
Comment (wilhelmklopp) [inline backend/uclapi/roombookings/management/commands/create_rooms.py]: what is this file?

**Comment by ChristopherHammond13** [comment]
Closing this as we are unlikely to cache data in this way. We'll probably unify data caching for `roombookings` and `timetable` in the future, but as `equipment` and `rooms` loads quickly enough without a cache we probably won't do that.

**Pull request closed without merging**